tensorflow/python/data/util/random_seed_test.py

```diff
@@ -18,64 +18,120 @@
+      ),
+      (
+          "CASE_8",
+          lambda: (0, constant_op.constant(0, dtype=dtypes.int64, name='zero')),
+          lambda: (0, 2**31 - 1)  # Avoid nondeterministic (0, 0) output
+      ),
+      (
+          "CASE_9",
+          lambda: (2**31 - 1, constant_op.constant(0, dtype=dtypes.int64, name='zero')),
+          lambda: (0, 2**31 - 1)  # Don't wrap to (0, 0) either
+      ),
+      (
+          "CASE_10",
+          lambda: (0, constant_op.constant(2**31 - 1, dtype=dtypes.int64, name='intmax')),
+          lambda: (0, 2**31 - 1)  # Wrapping for the other argument
+      )
+  ]
+  def reduce_fn(x, y):
+    name, input_fn, output_fn = y
+    return x + combinations.combine(
+        input_fn=combinations.NamedObject(
+            "input_fn.{}".format(name), input_fn),
+        output_fn=combinations.NamedObject(
+            "output_fn.{}".format(name), output_fn)
+    )
+
+  return functools.reduce(reduce_fn, cases, [])
+
+class RandomSeedTest(test_base.DatasetTestBase, parameterized.TestCase):
 
-  @test_util.run_in_graph_and_eager_modes
-  def testRandomSeed(self):
-    zero_t = constant_op.constant(0, dtype=dtypes.int64, name='zero')
-    one_t = constant_op.constant(1, dtype=dtypes.int64, name='one')
-    intmax_t = constant_op.constant(
-        2**31 - 1, dtype=dtypes.int64, name='intmax')
-    test_cases = [
-        # Each test case is a tuple with input to get_seed:
-        # (input_graph_seed, input_op_seed)
-        # and output from get_seed:
-        # (output_graph_seed, output_op_seed)
-        ((None, None), (0, 0)),
-        ((None, 1), (random_seed.DEFAULT_GRAPH_SEED, 1)),
-        ((1, 1), (1, 1)),
-        ((0, 0), (0, 2**31 - 1)),  # Avoid nondeterministic (0, 0) output
-        ((2**31 - 1, 0), (0, 2**31 - 1)),  # Don't wrap to (0, 0) either
-        ((0, 2**31 - 1), (0, 2**31 - 1)),  # Wrapping for the other argument
-        # Once more, with tensor-valued arguments
-        ((None, one_t), (random_seed.DEFAULT_GRAPH_SEED, 1)),
-        ((1, one_t), (1, 1)),
-        ((0, zero_t), (0, 2**31 - 1)),  # Avoid nondeterministic (0, 0) output
-        ((2**31 - 1, zero_t), (0, 2**31 - 1)),  # Don't wrap to (0, 0) either
-        ((0, intmax_t), (0, 2**31 - 1)),  # Wrapping for the other argument
-    ]
-    for tc in test_cases:
-      tinput, toutput = tc[0], tc[1]
-      random_seed.set_random_seed(tinput[0])
-      g_seed, op_seed = data_random_seed.get_seed(tinput[1])
-      g_seed = self.evaluate(g_seed)
-      op_seed = self.evaluate(op_seed)
-      msg = 'test_case = {0}, got {1}, want {2}'.format(
-          tinput, (g_seed, op_seed), toutput)
-      self.assertEqual((g_seed, op_seed), toutput, msg=msg)
-      random_seed.set_random_seed(None)
+  @combinations.generate(
+      combinations.times(
+          test_base.default_test_combinations(),
+          _test_random_seed_combinations()
+      )
+  )
+  def testRandomSeed(self, input_fn, output_fn):
 
-    if not context.executing_eagerly():
```

Comment:
Where is the equivalent of L67 through L78 being tested now?

Reply:
covered it in a new test now. Somehow missed adding this.

Reply:
the newly created test has no assertion, so it is not equivalent to the remove code

Reply:
@jsimsa the assert has been moved to the common `_evaluate()` method for both the tests.

Reply:
I see. The name of the `_evaluate` method mislead me. Consider renaming the method to `_checkEqual` or `_assertEqual`

Reply:
addressed in the commit. Used `_checkEqual` method instead of `_evaluate`